rpmio/rpmpgp.c

```diff
@@ -444,7 +444,7 @@ static int pgpPrtSubType(const uint8_t *h, size_t hlen, pgpSigType sigtype,
     int rc = 0;
 
     while (hlen > 0 && rc == 0) {
-	int impl = 0;
+	int recognized = 1;
```

Comment:
This makes new values default to being recognized, which is unsafe.

Reply:
A new value would get zeroed in the default case of the switch below, but I totally agree it's safer and far more obvious to have the default be non-recognized.